apps/extension/src/routes/popup/approval/approve-deny.tsx

```diff
@@ -16,8 +16,12 @@ export const ApproveDeny = ({
 
   return (
     <div
-      className='flex flex-row justify-between gap-4 rounded-md bg-black p-4 shadow-md'
-      style={{ paddingBottom: '28px', paddingTop: '28px' }}
+      className='flex flex-row justify-between gap-4 rounded-md p-4 shadow-md'
+      style={{
+        backgroundColor: '#1A1A1A',
+        paddingBottom: '28px',
+        paddingTop: '28px',
+      }}
```

Comment:
blending the colors of the layer containing the approve and deny buttons

-------

_before_

<img width="512" alt="Screenshot 2024-11-24 at 9 45 58 PM" src="https://github.com/user-attachments/assets/028e01a9-3f3d-42bc-903c-25b65bc49487">
<img width="512" alt="Screenshot 2024-11-24 at 9 47 19 PM" src="https://github.com/user-attachments/assets/247af4ef-82fd-4ae5-aa4f-c8ef64742a11">


---------------

_after_

<img width="512" alt="Screenshot 2024-11-24 at 9 42 50 PM" src="https://github.com/user-attachments/assets/93baadbb-56d7-448f-a539-a08ade8600c2">

<img width="512" alt="Screenshot 2024-11-24 at 9 42 38 PM" src="https://github.com/user-attachments/assets/a7e3ea37-ae96-4360-9097-49d7e964c1e2">